development/RELEASING.md

```diff
@@ -1,30 +1,31 @@
-# Releasing Guide
+# Release guide
 
-This guide explains how to release the plugin.
+This guide explains how to release the Orka Jenkins plugin.
```

Comment:
`nit` - we are inside the Orka Jenkins plugin repo, so repeating `Orka Jenkins` makes it a bit verbose